pkg/reconciler/knativeserving/common/extensions.go

```diff
@@ -32,7 +32,7 @@ func (platforms Platforms) Transformers(kubeClientSet kubernetes.Interface, inst
 		mf.InjectOwner(instance),
 		mf.InjectNamespace(instance.GetNamespace()),
 		ConfigMapTransform(instance, log),
-		DeploymentTransform(instance, log),
+		ResourceTransform(instance, log),
```

Comment:
I think name this is a little vague. DaemonSets and StatefulSets are really just specialized Deployments. And pretty much everything is a Resource. I'd say just keep the name `DeploymentTransform` or maybe `PodSpecableTransform`? Or save that for another PR?

Reply:
Sure i will keep `PodSpecableTransform`  :+1: